src/components/player/index.vue

```diff
@@ -461,6 +461,16 @@ export default {
 		},
 		loggedIn() {
 			this.buildTray();
+		},
+		radioType() {
+			this.buildTray();
+			if (this.playing) {
+				this.togglePlaying();
```

Comment:
Why toggle twice? To stop playing one type, and start playing the other type?

Reply:
Correct. I don't know how to do it otherwise, suggestions appreciated.